data/ui/window.ui

```diff
@@ -18,13 +18,15 @@
               <object class="GtkButton">
                 <property name="icon-name">list-add-symbolic</property>
                 <property name="action-name">app.new-note</property>
-                <property name="tooltip-text" translatable="yes">Create a new note</property>
+                <property name="tooltip-text" translatable="yes">New Note</property>
```

Comment:
again not sure about this. how about we keep it as is, then discuss further about this wording?? maybe you can link apps which use this kind of wording
```suggestion
                <property name="tooltip-text" translatable="yes">Create a new note</property>
```

Reply:
> maybe you can link apps which use this kind of wording

Yes: https://developer.gnome.org/hig/patterns/controls/buttons.html

> Button labels should use imperative verbs, using header capitalization. For example, Save or Update.

> Button labels should be short, in order to keep the button width narrow. Consider how labels will change width when localized.

Reply:
https://developer.gnome.org/hig/patterns/feedback/tooltips.html

> Tooltip labels should be short and written in header capitalization

HIG provides [some examples](https://developer.gnome.org/hig/patterns/feedback/tooltips.html?highlight=example) of good tooltips.

Reply:
What about "Add Note"?

Reply:
I changed it to "Create New Note".